sympy/tensor/functions.py

```diff
@@ -61,3 +63,46 @@ def __getitem__(self, index):
             arg.__getitem__(tuple(next(index) for i in shp))
             for arg, shp in zip(self.args, self._get_args_shapes())
         )
+
+
+def shape(expr):
+    """
+    Return the shape of the tensor.
```

Comment:
I think that it would be better to say "Return the shape of the matrix or tensor as a tuple"

Reply:
Actually I think maybe `array` is better than `tensor` here.